k2/csrc/CMakeLists.txt

```diff
@@ -2,8 +2,6 @@
 # The header should be included as `#include "k2/csrc/.."`, to avoid conflicts.
 include_directories(${CMAKE_SOURCE_DIR})
 
-add_subdirectory(host)
```

Comment:
And why remove `host`? Actually I want to keep `host` and `python` for now. When we create tests for those Fsa-related algorithms with new Ragged interface, I may reuse those (test) code (at least test cases) in `python` and `host`, it's better if we can keep them compiled all the time.

Reply:
got it, added it back